lib/ui.dart

```diff
@@ -68,4 +55,16 @@ class AppComponent {
 
     driver(client, users, process);
   }
+
+  addUser(userName) {
+    users = []..addAll(users)..add(userName);
```

Comment:
`users` is an `Iterable` so you can't add to it...I'll push some changes to an alternative solution, still not sure about it 😅